### ISSUE TITLE
Documentation copied from rusqlite is wrong

### ISSUE THREAD

**Issue by hkratz** [duckdb/duckdb-rs#7]
E.g. https://github.com/wangfenjin/duckdb-rs/blob/main/src/types/mod.rs 

Duckdb uses distinct integer types.

**Comment by wangfenjin** [comment]
yes, thanks for point out. Still working on the types as actually duckdb is static typing but sqlite is dynamic, they are totally different.

contribute are welcomed!

related to #4
